apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-11-17T22:06:28Z"
-  build_hash: 7cd74c3a855e38ea3fb5e1cf40b26fbe96283870
+  build_date: "2022-12-02T17:17:38Z"
+  build_hash: 560c702f71f3f9041b97417e84076d12721d9629
   go_version: go1.19
-  version: v0.19.3-15-g7cd74c3-dirty
-api_directory_checksum: f01a03847a7bb58e935fdca90ecfb910f645ea61
+  version: v0.19.3-9-g560c702
```

Comment:
Maybe time to use code-generator v0.20.1? `git fetch --tags --all` and rebasing will help

Reply:
Done.